deploy/09_set_minimum_stake.ts

```diff
@@ -0,0 +1,23 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { execute } = deployments
+  const { log } = deployments
+
+  const { deployer } = await getNamedAccounts()
+
+  const minStakeAmount = "39999999999999999999999"
```

Comment:
Why do we want to set the minimum stake to 39999999999999999999999? With dApp team owning T token contract from `dapp-development-goerli` deployment, all devs should be able to mint any amount of T they want.

Reply:
`39999999999999999999999` is the current min stake on mainnet that's why I picked this number but we could change to any number. We want to set the min stake to test the min stake validation in T dapp.